packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegateSubscriberRepository.swift

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+private var subscribers = [ExpoAppDelegateSubscriberProtocol]()
+private var reactDelegateHandlers = [ExpoReactDelegateHandler]()
+
+/**
+ Class responsible for managing access to app delegate subscribers and react delegates.
+ It should be used to access subscribers without depending on the `Expo` package where they are registered.
+ */
+@objc(EXExpoAppDelegateSubscriberRepository)
+public class ExpoAppDelegateSubscriberRepository: NSObject {
+  @objc
+  public static func getAllSubscribers() -> [ExpoAppDelegateSubscriberProtocol] {
+    return subscribers
+  }
```

Comment:
```suggestion
  @objc
  public static var subscribers(): [ExpoAppDelegateSubscriberProtocol] {
    return _subscribers
  }
```

maybe we can use computed properties and it would be clear from swift integration.
this is generated objc code fyi

```objc
SWIFT_CLASS_NAMED("ExpoAppDelegateSubscriberRepository")
@interface EXExpoAppDelegateSubscriberRepository : NSObject
SWIFT_CLASS_PROPERTY(@property (nonatomic, class, readonly, copy) NSArray<id <EXAppDelegateSubscriberProtocol>> * _Nonnull subscribers;)
+ (NSArray<id <EXAppDelegateSubscriberProtocol>> * _Nonnull)subscribers SWIFT_WARN_UNUSED_RESULT;
```